app/sidekiq/form526_status_polling_job.rb

```diff
@@ -50,28 +50,32 @@ def handle_response(response)
   end
 
   def handle_submission(status, form_submission)
+    submission_id = form_submission.id
+
     if %w[error expired].include? status
-      log_result('failure')
+      log_result('failure', submission_id)
```

Comment:
Would it be more helpful to log 'error' or 'expired' or is 'failure' good enough?

Reply:
Fair question. Probably, for the initial intended implementation, 'failure' met the stated needs, incrementing metrics based on submission state. Now, logging submission failure, regardless of reason, will at least notify that action is needed. 

This logging implemented to facilitate monitoring should only be temporarily needed, however, until the `Form526SubmissionFailureEmailJob` is enabled, as the failure would no longer be silent with the associated Veteran being notified of an issue submitting their claim.

It would still be possible to later call the endpoint with the submission's `backup_submitted_claim_id`, if one should need to know why the submission's state was set to 'rejected'. 

I'll let @SamStuckey provide any opinions he may have.